pymc3/backends/base.py

```diff
@@ -447,10 +463,13 @@ def get_sampler_stats(self, varname, burn=0, thin=1, combine=True,
                    for chain in chains]
         return _squeeze_cat(results, combine, squeeze)
 
-    def _slice(self, idx):
+    def _slice(self, slice):
         """Return a new MultiTrace object sliced according to `idx`."""
```

Comment:
according to `idx` --> according to `slice`

Reply:
fixed